pontoon/base/templates/translate.html

```diff
@@ -326,7 +326,13 @@ <h3 class="loading"><div class="fa fa-refresh fa-spin"></div></h3>
           <button id="clear" title="Clear Translation (Ctrl + Shift + Backspace)">Clear</button>
           <button id="save" title="Save Translation (Enter)"></button>
         {% else %}
-          <p id="sign-in-required"><a id="sidebar-signin" href="{{ provider_login_url(request) }}">Sign in</a> to translate.</p>
+          <p id="sign-in-required">
+            {% if settings.DJANGO_LOGIN %}
+              <a id="sidebar-signin" href="{{ url('standalone_login') }}">Sign in</a> to translate.
+            {% else %}
+              <a id="sidebar-signin" href="{{ provider_login_url(request) }}">Sign in</a> to translate.
+            {% endif %}
```

Comment:
To avoid duplicating code, I suggest you do something like this:

```
        {% if settings.DJANGO_LOGIN %}
          {% set login_url = url('standalone_login') %}
        {% else %}
          {% set login_url = provider_login_url(request) %}
        {% endif %}
        <a id="sidebar-signin" href="{{ login_url }}">Sign in</a> to translate.
```